packages/experimental/Link/src/useLink.ts

```diff
@@ -65,6 +66,8 @@ export const useLink = (props: LinkProps): LinkInfo => {
       ...rest,
       ...onKeyUpProps,
       ...pressable.props, // allow user key events to override those set by us
+      onKeyDown: Platform.OS === (('win32' as any) || 'windows') ? onKeyDown : undefined,
+      onMouseEnter: Platform.OS === (('win32' as any) || 'windows') ? pressable.props.onMouseEnter : undefined,
```

Comment:
Similar comment here

Reply:
Also `onKeyUp` here, since the spread of `onKeyUpProps` on line 67 is probably passing that in.